integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -0,0 +1,61 @@
+"""
+Transmits consenting enterprise learner data to the integrated channels.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import ugettext as _
+from . import IntegratedChannelCommandMixin, celery_task, INTEGRATED_CHANNEL_CHOICES
+
+
+class Command(IntegratedChannelCommandMixin, BaseCommand):
+    """
+    Management command which transmits learner course completion data to the IntegratedChannel(s) configured for the
+    given EnterpriseCustomer.
+
+    Collect the enterprise learner data for enrollments with data sharing consent, and transmit each to the
+    EnterpriseCustomer's configured IntegratedChannel(s).
+    """
+    help = _('''
+    Transmit Enterprise learner course completion data for the given EnterpriseCustomer.
+    ''')
+
+    def add_arguments(self, parser):
+        """
+        Add required --api_user argument to the parser.
+        """
+        parser.add_argument(
+            '--api_user',
+            dest='api_user',
+            default=None,
+            metavar=_('LMS_API_USERNAME'),
+            help=_('Username of a user authorized to fetch grades from the LMS API.'),
+        )
+        super(Command, self).add_arguments(parser)
+
+    def handle(self, *args, **options):
+        """
+        Transmit the learner data for the EnterpriseCustomer(s) to the active integration channels.
+        """
+        # Ensure that we were given an api_user name, and that User exists.
+        api_user = options['api_user']
+        if not api_user:
+            raise CommandError('Error: argument --api_user is required')
```

Comment:
@e-kolpakov There totally is:  `required=True`.  But I couldn't get it to work on my local devstack!  And I'd worn a dent in the wall trying to figure out why, so I just let it go.

But I'll try again once the rest of the issues are resolved.